docs/index.rst

```diff
@@ -112,6 +112,7 @@ Get Started
 
    admin/deployment/deployment_practices
    admin/deployment/landing_page.rst
+   admin/deployment/onion_name.rst
```

Comment:
Note that there are two places where we're linking to onion name content on SecureDrop.org:
- in the training schedule (to the blog post)
- in the landing page guidelines (to the FAQ)

I suggest we update both of these to point to the internal docs. In addition, are there other places where we should mention this new section?

Reply:
I updated the links in the two places you described. I can't think of any other places where it might be beneficial to add those links currently, although I can see where it might be useful if we decide to incorporate more of the support documentation in with the main documentation in the future. For now, if you think of any other places where it makes sense, please let me know and I'll get it added.